src/sounds_generation.py

```diff
@@ -14,16 +14,23 @@ def frequency_to_note(frequency):
     return round(12*math.log2(frequency/440.0)+45) % 12
 
 
+def note_to_frequency(note):
+    if note is None:
+        return 0
+    return pow(2, (note-9.0)/12.0)*440.0
+
+
 def get_sounds_from_file(file):
-    sounds = []
     snd = parselmouth.Sound(str(file))
     pitch = snd.to_pitch()
     frequencies = pitch.selected_array['frequency']
     notes = [frequency_to_note(freq) for freq in frequencies]
     return get_sounds_from_list(pitch.xs(), notes)
-    
+
 
 def get_sounds_from_list(timestamps, notes):
+    # for each timeframe recognise note and
+    # merge same notes together
```

Comment:
Merging is performed before, timestamps here are already merged. This comment is misleading